static_src/actions/org_actions.js

```diff
@@ -26,7 +27,14 @@ const orgActions = {
       orgGuid
     });
 
-    return cfApi.fetchOrg(orgGuid).then(orgActions.receivedOrg);
+    return cfApi.fetchOrg(orgGuid)
+      .then(orgActions.receivedOrg)
+      .catch((err) =>
+        errorActions.importantDataFetchError(
+          err,
+          'organization data may be incomplete'
+        )
```

Comment:
Just a minor thing, but it's worth adding a helper to errorActions like, `defaultFetchErrorHandler` or something that returns a function that calls the action. Then you could have:
```javascript
    return cfApi.fetchOrg(orgGuid)
      .then(orgActions.receivedOrg)
      .catch(errorActions.defaultFetchErrorHandler('organization data may be incomplete'));
```